libraries/USB/src/USBMIDI.h

```diff
@@ -0,0 +1,72 @@
+
```

Comment:
Include the `social_caps.h` and add a check also to header file.

```suggestion
#include "soc/soc_caps.h"
#if SOC_USB_OTG_SUPPORTED
```